src/datumaro/plugins/data_formats/coco/page_mapper.py

```diff
@@ -81,10 +84,20 @@ def to_dict(self):
 class COCOSection(Enum):
     IMAGES = b'"images"'
     ANNOTATIONS = b'"annotations"'
+    CATEGORIES = b'"categories"'
+
+    @property
+    def is_necessary(self) -> bool:
+        """Flat to indicate whether the section is necessary or not"""
+        if self in {COCOSection.IMAGES, COCOSection.ANNOTATIONS}:
+            return True
+
+        return False
 
 
 class FileReaderWithCache:
-    _n_chars = 1024 * 1024 * 16  # 16MB
+    # _n_chars = 1024 * 1024 * 16  # 16MB
+    _n_chars = 1024 * 64  # 64KB
```

Comment:
The previous one (16MB) is overspec when reading?

Reply:
Yes, I found from my experiment that there are many cache misses (effort to cache > gain from cache) in the later part of the COCO2017 instance dataset. I decided it from the real-life experiment.